tracing-appender/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 0.2.3 (October 12, 2023)
+
+This release has a large number of changes to `tracing-appender`:
+
+### Added
+- **non_blocking**: -Name spawned worker thread [#2219]
+- **non_blocking**: Allow worker thread name to be configured [#2365]
+- **rolling**: `RollingFileAppender::builder` for configuring `RollingFileAppender`s [#2227]
+- **rolling**: `Builder::filename_suffix` method to configure the suffix for log files [#2225]
+- **rolling**: `Builder::max_log_files` method to enable automatically deleting old log files [#2323]
+
+### Documented
+
+- **rolling**: Clarify file appender docs [#2689]
+- **rolling**: Fix typo in RollingFileAppender docs [#2375]
```

Comment:
nit: can we put parens around the issue links _as well_, the way we do in the other changelog entries? sorry.
```suggestion
- **non_blocking**: -Name spawned worker thread ([#2219])
- **non_blocking**: Allow worker thread name to be configured ([#2365])
- **rolling**: `RollingFileAppender::builder` for configuring `RollingFileAppender`s ([#2227])
- **rolling**: `Builder::filename_suffix` method to configure the suffix for log files ([#2225])
- **rolling**: `Builder::max_log_files` method to enable automatically deleting old log files ([#2323])

### Documented

- **rolling**: Clarify file appender docs ([#2689])
- **rolling**: Fix typo in RollingFileAppender docs ([#2375])
```